tests/syft/lib/allowlist_test.json

```diff
@@ -0,0 +1,311 @@
+      "data_types": ["common"],
+      "tensors": ["tensor1", "tensor2"],
+      "inputs": [null]
+    },
+    "tensor_method_noinput": {
+      "description": "Covers most tensor method attributes with no input",
+      "property": false,
+      "return_type": "torch.Tensor",
+      "data_types": ["common"],
+      "tensors": ["tensor1", "tensor2"],
+      "inputs": [null]
+    },
+    "tensor_method_input": {
+      "description": "Covers most tensor method attributes",
+      "property": false,
+      "return_type": "torch.Tensor",
+      "data_types": ["common"],
+      "tensors": ["tensor1", "tensor2"],
+      "inputs": ["all"]
+    },
+    "tensor_float_division_method_v150": {
+      "description": "Skips running float / div methods which crash on torch==1.4.0",
+      "property": false,
+      "return_type": "torch.Tensor",
+      "data_types": ["common"],
+      "tensors": ["tensor1_nonzero", "tensor2_nonzero_rounded"],
+      "inputs": ["all_nonzero"],
+      "min_version": "1.5.0"
+    },
+    "tensor_bitwise_method_v150": {
+      "description": "Skips running bitwise methods not in torch==1.4.0",
+      "property": false,
+      "return_type": "torch.Tensor",
+      "data_types": ["common"],
+      "tensors": ["tensor1", "tensor2"],
+      "inputs": ["all"],
+      "min_version": "1.5.0"
+    },
+    "tensor_logical_method_v150": {
+      "description": "Skips running logical methods not in torch==1.4.0",
+      "property": false,
+      "return_type": "torch.Tensor",
+      "data_types": ["common"],
+      "tensors": ["tensor1", "tensor2"],
+      "inputs": ["self", [0], [1]],
+      "min_version": "1.5.0"
+    }
+  },
+  "tests": {
+    "torch.Tensor": {
+      "T": {
+        "profile": "tensor_property_noinput"
+      },
+      "t": {
+        "profile": "tensor_method_noinput"
+      },
+      "__abs__": {
+        "profile": "tensor_method_noinput",
+        "skip": [
+          { "data_types": ["bool"] },
+          { "data_types": ["float16", "bfloat16"], "max_version": "1.5.1" }
+        ]
+      },
+      "__add__": {
+        "profile": "tensor_method_input",
+        "skip": [{ "data_types": ["float16"], "max_version": "1.5.1" }]
+      },
+      "__and__": {
+        "profile": "tensor_method_input",
+        "data_types": ["bool", "integer"],
+        "skip": [
+          {
+            "data_types": ["bool"],
+            "inputs": [0, 1],
+            "min_version": "1.5.0",
+            "max_version": "1.6.0"
+          }
+        ]
+      },
+      "__div__": {
+        "profile": "tensor_float_division_method_v150",
+        "skip": [
+          { "data_types": ["bool"], "max_version": "1.5.1" },
+          { "data_types": ["float16"], "max_version": "1.5.1" },
+          { "min_version": "1.6.0" }
+        ]
+      },
+      "__floordiv__": {
+        "profile": "tensor_float_division_method_v150",
+        "skip": [
+          { "data_types": ["bool"], "max_version": "1.6.0" },
```

Comment:
Yes, i'm still getting confused by these names myself, especially since the condition itself is a kind of negative so it gets into the realm of double negatives etc.

Reply:
I have changed these to lte_version and gte_version lets see if that makes more sense.